main.tf

```diff
@@ -299,7 +299,6 @@ resource "null_resource" "reset_api_key" {
 ##############################################################################
 
 data "ibm_container_cluster_config" "cluster_config" {
-  count             = var.enable_ocp_console || var.verify_worker_network_readiness || lookup(var.addons, "cluster-autoscaler", null) != null ? 1 : 0
```

Comment:
There still needs to be a count here, but it needs to be updated to so it executes if `enable_ocp_console` is true OR flase

Reply:
`count = var.enable_ocp_console != null || var.verify_worker_network_readiness || lookup(var.addons, "cluster-autoscaler", null) != null ? 1 : 0`